deno_dist/request.ts

```diff
@@ -58,7 +58,7 @@ export function extendRequestPrototype() {
     if (this.paramData) {
       if (key) {
         const param = this.paramData[key]
-        return param ? decodeURIComponent(param) : undefined
```

Comment:
I think the previous code is fine. We just need to make the type for when it's undefined be `string | undefined` or `string | null`.